src/triangulum/systemd.clj

```diff
@@ -28,28 +29,32 @@ PrivateTmp=true
 WantedBy=default.target
 "))
 
-(defn- enable-systemd [{:keys [repo http https dir extra-aliases]}]
+(defn fmt-service-file [{:keys [repo-dir http https extra-aliases]}]
+  (format unit-file-template
+          repo-dir
+          (or extra-aliases "")
+          (if http (str "-p " http) "")
+          (if https (str "-P " https) "")))
+
+(defn- enable-systemd [{:keys [repo dir] :as config}]
   (let [service-name (str "cljweb-" repo)
         full-dir     (-> dir
-                         (io/file)
-                         (.getAbsolutePath)
+                         (fs/expand-home)
+                         (fs/absolutize)
                          (remove-end "."))
-        repo-dir     (if (= (.getName (io/file full-dir)) repo)
+        repo-dir     (if (= (fs/file-name full-dir) repo)
                        full-dir
                        (-> full-dir
                            (end-with "/")
                            (str repo)))
-        unit-file    (str user-systemd-path service-name ".service")]
-    (if (.exists (io/file repo-dir "deps.edn"))
+        service-file (str service-name ".service")
+        unit-file    (path user-systemd-path service-file)]
+    (if (fs/exists? (io/file repo-dir "deps.edn"))
       (do
-        (io/make-parents unit-file)
-        (spit unit-file
-              (format unit-file-template
-                      repo-dir
-                      (or extra-aliases "")
-                      (if http (str "-p " http) "")
-                      (if https (str "-P " https) "")))
-        (io/copy (io/file unit-file) (io/file (str user-home "/.config/systemd/user/" unit-file)))
+        (fs/create-dirs user-systemd-path)
+        (fs/create-dirs systemd-multi-user-path)
+        (spit unit-file (fmt-service-file (merge config {:repo-dir repo-dir})))
+        (fs/create-sym-link (path systemd-multi-user-path service-file) unit-file)
```

Comment:
According to the [Systemd docs](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#:~:text=in%20version%20233.-,enable,-UNIT%E2%80%A6%2C), `systemctl --user enable` step will add a set of symlinks as laid out in the `[Install]` section of the service file. 

I think if we changed `unit-file-template` to be `WantedBy=multi-user.target` instead of `WantedBy=default.target` then we can get rid of this line because `(shell-wrapper shell-opts user-systemctl "enable" service-name)` will take care of this for us.